python/lsst/ip/diffim/imageMapReduce.py

```diff
@@ -523,7 +563,7 @@ def _reduceImage(self, mapperResults, exposure, **kwargs):
         result = self.reducerSubtask.run(mapperResults, exposure, **kwargs)
         return result
 
-    def _generateGrid(self, exposure, forceEvenSized=False):
+    def _generateGrid(self, exposure, forceEvenSized=False, **kwargs):
```

Comment:
That is noted in a comment where this function is called (line 528) but will add a comment here.